src/components/Card/Card.tsx

```diff
@@ -1,15 +1,11 @@
-import styles from "./Card.module.scss"
+import styles from "./Card.module.scss";
 
 interface Content {
-    children: any
+  children: JSX.Element | string | JSX.Element[];
 }
 
 const Card = (props: Content) => {
```

Comment:
```suggestion
const Card = (props: Content) => (
    <div className={styles.card}>
        {props.children}
    </div>
)
```

We do not have any logic in this component therefore we can use this shortcut. Add this in other suitable components as well.